src/message/MessageContainer.js

```diff
@@ -37,7 +37,7 @@ export default class MessageContainer extends React.PureComponent {
         doNarrow={doNarrow}
         onPress={this.handlePress}
       >
-        {renderHtmlChildren({ childrenNodes: dom, auth })}
+        {renderHtmlChildren({ childrenNodes: dom, auth, doNarrow })}
```

Comment:
nice, I have factored out the onPress behavior to messageContainer and htmlATag just uses this  `onPress`.